packages/deployments/contracts/contracts/messaging/connectors/zksync/ZkSyncHubConnector.sol

```diff
@@ -129,6 +123,7 @@ contract ZkSyncHubConnector is HubConnector, GasCap {
       processed[_root] = true;
       // update the root on the root manager
       IRootManager(ROOT_MANAGER).aggregate(MIRROR_DOMAIN, _root);
+      emit MessageSent(_message, bytes(""), msg.sender);
```

Comment:
We need to emit emit `MessageProcessed(_message, msg.sender)`

Reply:
fixed! https://github.com/connext/nxtp/pull/2493/commits/13f7262040d36c71f1b84ee5565a4d4207889f23